File: lib/src/main/java/com/hammwerk/wizardpager/SingleFixedChoicePage.java
package com.hammwerk.wizardpager;

import android.support.v4.app.Fragment;

import com.hammwerk.wizardpager.core.Page;

public class SingleFixedChoicePage extends Page<Integer> {
	private String[] choices;

	public SingleFixedChoicePage(String title) {
		this(title, true);
	}

	public SingleFixedChoicePage(String title, boolean required) {
		super(title, required);
	}

	@Override
	protected Fragment createFragment() {
		return SingleFixedChoicePageFragment.createInstance(this, choices);
	}

	public SingleFixedChoicePage setChoices(String... choices) {
		this.choices = choices;
		return this;
	}
}

File: lib/src/main/kotlin/com/hammwerk/wizardpager/SingleFixedChoicePage.kt
package com.hammwerk.wizardpager

import com.hammwerk.wizardpager.core.Page

class SingleFixedChoicePage @JvmOverloads constructor(title: String, vararg val choices: String,
													  required: Boolean = true) :
		Page(title, SingleFixedChoicePageFragment::class.qualifiedName!!, required) {
	var selectedChoice: Int? = null
}
